pages/press/index.js

```diff
@@ -0,0 +1,348 @@
+                It’s a large company with 100,330 employees.
+              </Paragraph>
+            </li>
+            <li className="py-1">
+              <Paragraph typeStyle="body-md" className="my-4">
+                <TextLink href="https://www.iqvia.com/" target="_blank" className="no-underline">
+                  <b>IQVIA</b>
+                </TextLink>: Company providing services for the combined health information technology and clinical research industries. 
+                It employs more than 58.000 people in over 100 countries
+              </Paragraph>
+            </li>
+          </ul>
+        </Paragraph>
+
+        <Heading level="h3" typeStyle="heading-md" className="my-5">
+          <span id="funding">Funding</span>
+        </Heading>
+        <Paragraph typeStyle="body-md" className="my-4">
+          AsyncAPI is an open source project and is therefore funded by talent and voluntary contributions. 
+          It relies on the expertise and dedication of the <b>Community developers</b> who believe in the project and contribute to its evolution.
+        </Paragraph>
+        <Paragraph typeStyle="body-md" className="my-4">
+          Financially, there are individual donations and contributions made by users of the project, and others that come from different companies and initiatives that trust and support AsyncAPI through sponsorship. 
+          The main sponsors that contribute to sustain this initiative are Postman, IQVIA Technology, Mulesoft, Salesforce, SAP, SLACK, Solace, and Tibco.
+        </Paragraph>
+        <Paragraph typeStyle="body-md" className="my-4"></Paragraph>
+        <Paragraph typeStyle="body-md" className="my-4">
+          All the information about the project's economy, the amount of the donations, the identity of the donors, the sponsors supporting the project and the use of this money is public. 
+          99% of the funding goes through the Open Collective. <TextLink href="https://opencollective.com/asyncapi" target="_blank">More information here</TextLink>.
+        </Paragraph>
+        <Paragraph typeStyle="body-md" className="my-4">
+          Open Collective is an online crowdfunding platform for open and transparent communities. 
+          It provides the necessary tools to collect and share the finances raised with complete transparency. 
+          It is an ideal collective for communities such as collaborative groups or open source projects among others. 
+          This platform can be used to raise and transparently disburse money for the benefit of their members, their registered projects, and those who want to support them.
+        </Paragraph>
+        <Paragraph typeStyle="body-md" className="my-4">
+          GitHub is a project management platform that also has the option to manage part of the financing. 
+          It is also important that the company making the largest contribution to the initiative is Postman.
```

Comment:
This line sounds... odd... I don't think it makes sense to have this sentence here, I would actually remove it.